test.py

```diff
@@ -0,0 +1,3 @@
+I have put some things in here
+
+and also on this line.
```

Comment:
I already have this line in another code file that I created earlier, and this is ambiguous.